packages/design-system/src/utils/index.js

```diff
@@ -21,5 +21,13 @@ export { default as useLiveRegion } from './useLiveRegion';
 export { default as uniqueEntriesByKey } from './uniqueEntriesByKey';
 export { default as deepMerge } from './deepMerge';
 export { default as useMouseDownOutsideRef } from './useMouseDownOutsideRef';
+export { default as pointerEventsCss } from './pointerEventsCss';
+export { default as objectWithout } from './objectWithout';
+export { default as getKeyboardMovement } from './getKeyboardMovement';
+export { default as areEventsDragging } from './areEventsDragging';
```

Comment:
This also feels a bit out of place here 🤔 This util was mostly created for Moveable -- since a click for moveable triggers dragging but we want to be able to handle click separately. So in this sense, it might make sense for the Moveable package. However, it's used in one more place but maybe that's ok.

Reply:
Both `element-library` and `story-editor` depend on `moveable` already, so it should be fine to move it to that package.